firestore.rules

```diff
@@ -7,12 +7,12 @@ service cloud.firestore {
 
     function isGlobalAdmin(email) {
       let adminUser = get(/databases/$(database)/documents/admins/$(email)).data;
-      return adminUser.is_global_admin;
+      return 'is_global_admin' in adminUser && adminUser.is_global_admin;
```

Comment:
what does this change do?

Reply:
Makes sure the function doesn't crash if there is no key called `is_global_admin`.